Sources/NIO/Selector.swift

```diff
@@ -549,7 +549,7 @@ internal class Selector<R: Registration> {
                 // Consume event
                 var val: UInt64 = 0
                 // We are not interested in the result
-                _ = Glibc.read(self.timerFD, &val, MemoryLayout.size(ofValue: val))
+                _ = try! Posix.read(self.timerFD, &val, MemoryLayout.size(ofValue: val))
```

Comment:
Looks like you're missing some labels here: https://github.com/apple/swift-nio/blob/546a84c4e11b8100b962c55afc77f7c3f94d1d2c/Sources/NIO/System.swift#L396

Reply:
Thanks for checking. I am figuring out the values.